examples/example_metadata/geogrid--facies.yml

```diff
@@ -36,7 +36,7 @@ display:
   name: facies
 file:
   absolute_path: /some/absolute/path/
-  checksum_md5: d205770796667096bac52031c0c6c421
+  checksum_md5: dcb63dea7fee7baaae6e0c04d80317b1
```

Comment:
and this is just annoying.. this happens due to the timestamp added by roffio 🙃
and we haven't seen it before since I just added roff files to the examples..